src/pu-emmc.c

```diff
@@ -512,6 +514,42 @@ pu_emmc_init(G_GNUC_UNUSED PuEmmc *self)
 {
 }
 
+static PedSector
+pu_emmc_get_partition_table_size(PuEmmc *emmc)
+{
+    if (g_str_equal(emmc->disktype->name, "msdos"))
+        return PARTITION_TABLE_SIZE_MSDOS;
+    else if (g_str_equal(emmc->disktype->name, "gpt"))
+        return PARTITION_TABLE_SIZE_GPT;
+    return 1;
+}
+
+static gboolean
+pu_emmc_get_input_size(PuEmmcInput *input,
+                       const gchar *prefix,
+                       GError **error)
+{
+    g_autoptr(GFile) file = NULL;
+    g_autoptr(GFileInfo) finfo = NULL;
+    g_autofree gchar *path = NULL;
+
+    g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
+
+    path = pu_path_from_uri(input->uri, prefix, error);
+    if (path == NULL)
+        return FALSE;
+
+    file = g_file_new_for_path(path);
+    finfo = g_file_query_info(file, G_FILE_ATTRIBUTE_STANDARD_SIZE,
+                                    G_FILE_QUERY_INFO_NONE, NULL, error);
```

Comment:
The indentation is too far here and should be aligned with the opening parenthesis.